indico/modules/events/registration/templates/management/_registration_details.html

```diff
@@ -179,6 +180,13 @@
                     {% trans %}Get ticket{% endtrans %}
                 </a>
             {% endif %}
+            {% if can_get_document %}
+                <a href="#" class="i-button accept icon-agreement"
+                data-callback="printReceipts"
+                data-params="{{ {'registration_id': [registration.id|string], 'event_id': registration.registration_form.event.id} | tojson | forceescape }}">
+                    {% trans %}Get document{% endtrans %}
```

Comment:
"Get" sounds weird, this is about generating documents

Reply:
`Generate document` sounds good?

Reply:
or... `Generate receipt` is more appropriate for this context?

Reply:
yes

Reply:
or... `Generate receipt` is more appropriate for this context?

Reply:
no, don't use "receipt" since it can be used for any kinds of documents, not just receipts.